src/seedu/addressbook/data/person/_SimilarNameTest.java

```diff
@@ -0,0 +1,32 @@
+package seedu.addressbook.data.person;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+public class _SimilarNameTest {
+
+	@Test
+	public void SameNameAndSpaces() throws IllegalValueException {
```

Comment:
According to the coding convention, the method name should start from which method you want to test, then what you want to test and finally expected return. Such as `isSimilar_sameNameAndSpaces_returnTrue`.